### ISSUE TITLE
Cannot install 30-Sept-2022 release in Delphi XE

### ISSUE THREAD

**Issue by JohnnyMarauder** [IndySockets/Indy#433]
Followed the directions to remove Indy 10 with the default XE install.  Cloned the repo to C:\Repos\Indy and then started Delphi.  Told Delphi to not load the various .dpks since Indy10 has been force deleted per the wiki.  Opened the Indy150.groupproj in the repo folder, and then went through the list to compile the .bpl files in the order listed.  

When I get to dclIndyCore150.bpl, I get the error that IdAboutVCL.pas(81): F1026 File not found: 'IdGlobal.dcu'  

Looks like this is not part of the list, or am I missing some library setup in XE that needs to be done before I start the compile and install?

John

**Comment by JohnnyMarauder** [comment]
![Screenshot 2022-10-02 093119](https://user-images.githubusercontent.com/3180543/193459507-ffa8fcac-50bd-469f-83ca-01046ceb995e.png)

**Comment by JohnnyMarauder** [comment]
Ok, once I added the three library paths  for Lib\Core, Lib\System and Lib\Protocols  it compiles and installs. Duh, not my finest moments today. LOL
